core/src/main/java/com/yahoo/imapnio/async/request/AbstractFolderActionCommand.java

```diff
@@ -33,26 +38,54 @@
     protected AbstractFolderActionCommand(@Nonnull final String op, @Nonnull final String folderName) {
         this.op = op;
         this.folderName = folderName;
+        this.qResyncParameter = null;
```

Comment:
Who else uses AbstractFolderActionCommand? Is it only select & examine. If status also extends from here, then you cannot add this here.

Reply:
Fixed